assembly-descriptors/pom.xml

```diff
@@ -4,7 +4,7 @@
 	<parent>
 		<groupId>org.eclipse.rdf4j</groupId>
 		<artifactId>rdf4j</artifactId>
-		<version>4.3.1-SNAPSHOT</version>
+		<version>4.3.1-GH4574-SNAPSHOT</version>
```

Comment:
You probably meant to clean up this before committing.

Reply:
i will change that. But it's always a problem to use the same version in feature branches. maybe rdf4j should think about using the maven gitflow plugin ?

Reply:
done!